docs/release-notes/8.16.asciidoc

```diff
@@ -1,6 +1,46 @@
 [[release-notes-header-8.16.0]]
 == 8.16
 
+[discrete]
+[[release-notes-8.16.6]]
+=== 8.16.6
+
+[discrete]
+[[known-issue-8.16.6]]
+==== Known issues
+
+// tag::known-issue[]
+[discrete]
+.Duplicate alerts can be produced from manually running threshold rules 
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running threshold rules could produce duplicate alerts if the date range was already covered by a scheduled rule execution.
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.Manually running custom query rules with suppression could suppress more alerts than expected
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running a custom query rule with suppression could incorrectly inflate the number of suppressed alerts. 
+====
+// end::known-issue[]
+
+[discrete]
+[[bug-fixes-8.16.6]]
+==== Bug fixes
+* Fixes a bug that affected column widths in Timeline ({kibana-pull}214178[#214178]).
+* Fixes a bug that prevented you from being able to save event filters ({kibana-pull}213805[#213805]).
+* Allows Automatic Import to handle a variety of structures when parsing system logs ({kibana-pull}212611[#212611]).
+* Allows data view selections to persist when you save a Timeline ({kibana-pull}211343[#211343]).
```

Comment:
Fixes a bug, preventing the selected data view / alerts only option from being reliably persisted in timeline

Reply:
```suggestion
* Fixes a bug that prevented the selected data view and **Show only detection alerts** option from being reliably persisted in Timeline ({kibana-pull}211343[#211343]).
```